include/ringmesh/geomodel/core/geomodel.h

```diff
@@ -399,6 +392,107 @@ namespace RINGMesh
         }
         SurfaceSide voi_surfaces() const;
 
+        /*!
+         * \name Transfers of attributes
+         * @{
+         */
+        template < typename AttributeType >
+        void transfer_cell_attribute_to_gm_regions(

```

Comment:
The signature of the function does not highlight where is stored the cell_attribute to transfer. From GMM?